packages/ecr-credential-provider/Cargo.toml

```
@@ -0,0 +1,20 @@
+[package]
+name = "ecr-credential-provider"
+version = "0.1.0"
+edition = "2018"
+publish = false
+build = "build.rs"
+
+[lib]
+path = "pkg.rs"
+
+[package.metadata.build-package]
+releases-url = "https://github.com/kubernetes/cloud-provider-aws/releases"
+
+[[package.metadata.build-package.external-files]]
+url = "https://github.com/kubernetes/cloud-provider-aws/archive/refs/tags/v1.25.1.tar.gz"
```

nit: I tend to prefer this form of the URL for consistency with other packages:
```suggestion
url = "https://github.com/kubernetes/cloud-provider-aws/archive/v1.25.1/cloud-provider-aws-1.25.1.tar.gz"
```

---

I wouldn't call it a nit, in the cache this will be named `1.25.1.tar.gz` which could collide with other files (if there are any).

---

To clarify, although not ideal, naming collisions in Bottlercoket's lookaside cache is not an issue since the files are prefixed with their hash in the bucket and buildsys knows how to retrieve the file based on its hash. 

We already have naming collisions between all the EKS-D source tarballs for the different K8s packages since they're all named `kubernetes-src.tar`

---

I was thinking - and I think @arnaldo2792 was thinking - of hypothetical collisions within the spec file, where prefixing the version with the source project's name makes it clear which "v1.25.1.tar.gz" is being unpacked.

---

Will change.

Out of curiosity, how did you get that path? If you go to the GitHub repo for the project and look at the release artifacts, the link that it provides is the one I had used. Is there somewhere else that publishes the more descriptively named link? Or did you just know GitHub also provides a link that way?